### PR TITLE
Set defaults for mailbox flags

### PR DESCRIPTION
**Pull request by nicodh**
resolves deltachat/deltachat-desktop#875

I'm not sure where the defaults "should" come from, but this fixes the issue...

### CODE REVIEW

File: test/integration.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ function configureDefaultDC (dc) {
 // 4. test opening an already configured account (re-open above)
 
 test('setUp dc context', t => {
-  t.plan(13)
+  t.plan(18)
```
Comment (ralphtheninja) [inline test/integration.js]: This should probably be `17`. It seems the integration tests never finish on e.g. AppVeyor.

Comment (ralphtheninja) [reply test/integration.js]: Also doesn't finish on Travis https://travis-ci.org/deltachat/deltachat-node/jobs/552376198

**Comment by ralphtheninja** [comment]
For some reason we get the link error on Travis/Mac. I have no idea why this happens. It used to work well but something has changed.

File: test/integration.js (hunk 1 above)
Comment (nicodh) [reply test/integration.js]: Hmm - strange in my local tests it worked. Will check again...

**Comment by ralphtheninja** [comment]
@nicodh Mind rebasing onto latest master? If tests pass then just merge this one.

**Pull request merged**